thunderfish/sortfishtype.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+import peakdetection as pkd
+from IPython import embed
```

Comment:
ok. I removed all the dependencies I saw. I agree this is not needed, but might come again during the hard developing stage we're going through. embed() is essential for debugging after all!